packages/account/src/Styles/account.scss

```diff
@@ -442,6 +442,21 @@ $MIN_HEIGHT_FLOATING: calc(
             border-top: 1px solid var(--general-active);
         }
 
+        &__authentication-message {
+            display: flex;
+            flex-direction: row;
```

Comment:
if I'm not mistaken, 'row' is a default value. no necessity to define it explicitly. Could you please check